packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -436,11 +434,23 @@ class SlideshowView extends React.Component {
     }
   };
 
+  getFirstIdx(itemIdx) {
+    const activeItem = this.props.galleryStructure.galleryItems[itemIdx];
+    const activeItemId = activeItem.itemId;
+    const firstItemInstance = this.props.galleryStructure.galleryItems.find((item) => item.itemId === activeItemId);
+    return firstItemInstance.idx;
+  }
```

Comment:
I think this is more elegant - 
```suggestion
getFirstIdx(itemIdx) {
	  const { galleryItems } = this.props.galleryStructure;
	  const activeItemId = galleryItems[itemIdx].itemId;
	  return galleryItems.find(item => item.itemId === activeItemId).idx;
	}
```